### PR TITLE
feat(docs): add generators for release posts

### PR DESCRIPTION
**Pull request by tknickman**
### Description

Use our own generators for release blog posts

### CODE REVIEW

File: docs/turbo/generators/utils.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const axios = require("axios");
```
Comment (mehulkar) [inline docs/turbo/generators/utils.js:1]: can we stop using axios for new code and just use native fetch or undici? 😢  (we need to upgrade to node 18 too, so maybe undici till that happens)

**Review by mehulkar — APPROVED** [review]
Can you add something in CONTRIBUTING.md or maybe docs/README.md on how to use the generator? I know I should know the API, but I probably won't remember :D

File: docs/turbo/generators/utils.js (hunk 1 above)
Comment (tknickman) [reply docs/turbo/generators/utils.js:1]: Yea fair we do need to rip it all out soon

**Comment by tknickman** [comment]
Docs added!

**Pull request merged**